common/src/cobs/issue.rs

```diff
@@ -472,6 +466,15 @@ mod events {
         ObjId,
     };
 
+    impl From<State> for String {
+        fn from(state: State) -> Self {
+            match state {
+                State::Open => "Open issue".to_owned(),
+                State::Closed { .. } => "Close issue".to_owned(),
+            }
+        }
+    }
```

Comment:
This will apply to *all* conversions from State to String, so it's not what we want. If you want something like this, I'd implement a private function on `State` called `lifecycle_message`